modules/redis/scanner.go

```diff
@@ -152,6 +169,71 @@ func (scan *scan) Close() {
 	defer scan.close()
 }
 
+func getUnmarshaler(file string) (func([]byte, interface{}) error, error) {
+	var unmarshaler func([]byte, interface{}) error
+	switch ext := filepath.Ext(file); ext {
+	case ".json":
+		unmarshaler = json.Unmarshal
+	case ".yaml", ".yml":
+		unmarshaler = yaml.Unmarshal
+	default:
+		err := fmt.Errorf("File type %s not valid.", ext)
+		return nil, err
+	}
+	return unmarshaler, nil
+}
+
+func getFileContents(file string, output interface{}) error {
+	unmarshaler, err := getUnmarshaler(file)
+	if err != nil {
+		return err
+	}
+	fileContent, err := ioutil.ReadFile(file)
+	if err != nil {
+		return err
+	}
+	err = unmarshaler([]byte(fileContent), output)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Initializes the command mappings
+func (scanner *Scanner) initCommands() error {
+	scanner.commandMappings = map[string]interface{}{
```

Comment:
Any reason this is `map[string]interface{}` instead of `map[string]string`?

Reply:
That was left over from a previous attempt implementation, so I'll change it to `map[string]string`

Reply:
Fixed!